runners/direct-java/build.gradle

```diff
@@ -70,17 +71,16 @@ dependencies {
   shadow library.java.vendored_grpc_1_26_0
   shadow library.java.joda_time
   shadow library.java.slf4j_api
-  shadow library.java.args4j
+  permitUnusedDeclared "org.apache.beam:beam-vendor-grpc-1_26_0:0.3"
+  permitUnusedDeclared project(":runners:java-fn-execution")
+  permitUnusedDeclared project(":sdks:java:fn-execution")
```

Comment:
These seem wrong. The direct runner should either use these or not depend on them.

Reply:
If we completely remove these project-project dependencies, Java Wordcount Direct Runner check if failing.

Reply:
@kennknowles , these dependency are indirectly being declared i.e there is transitive dependency for these projects. But the module does not require them, so we need to declare them as permitUnusedDeclared .

Reply:
Transitive dependencies are automatically included. So there is something else that makes these transitive dependencies not work. If the compile step succeeds but the test step does not succeed, that means it is a runtime dependency of the test. I think the only thing we can do is to add the exclusion, for now. Please add a comment that these are runtime dependencies of the tests.